run_desktop_tests.sh

```diff
@@ -24,6 +24,9 @@ cd libraries/crypto
 cargo fmt --all -- --check
 cd ../..
 
+echo "Building sha256sum tool..."
```

Comment:
I wouldn't bother updating the travis file anymore. It's going away soon (workflows seem more reliable and OS X build on travis proved it was flaky).

Reply:
I find it useful to actually run the tests on a desktop (hence the script's name) to iterate fast locally instead having to wait for CI to complete.
- Regarding Travis, we can just remove the `.travis.yml` if that's the issue.
- This script should ideally replicate what the GitHub workflows do.
  - One possibility is to move away from the action dependencies (e.g. `uses: actions-rs/cargo@v1`) which are hard to replicate locally, and instead just run plain Bash scripts in the workflow. This may have some drawbacks by removing the additional features that actions provide, but would allow consistency between local tests and workflows.
  - Another possibility is to opportunistically synchronize this script with the workflows (as is done in this pull-request). Although things can go out of sync, it's easy to re-sync, so I'm also fine with it.

Reply:
My bad, I thought it was the `travis.yml` that was updated. It's ok then